BaseClasses.py

```diff
@@ -813,10 +810,8 @@ def can_reach(self, state: CollectionState) -> bool:
 
         return False
 
-    def connect(self, region: Region, addresses: Any = None, target: Any = None) -> None:
+    def connect(self, region: Region) -> None:
```

Comment:
thoughts on how we should resolve that? pokemonRB already has a custom Entrance so it makes more sense to me for it to set these attrs on itself, rather than this being a core thing, since no other games use it. @Alchav ?